cernopendata/modules/theme/assets/semantic-ui/templates/search/utils.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * -*- coding: utf-8 -*-
+ *
+ * This file is part of CERN Open Data Portal.
+ * Copyright (C) 2021 CERN.
+ *
+ * CERN Open Data Portal is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * CERN Open Data Portal is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CERN Open Data Portal; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307, USA.
+ *
+ * In applying this license, CERN does not
+ * waive the privileges and immunities granted to it by virtue of its status
+ * as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+export const stripHtml = (input) => input?.replace(/<[^>]*>?/gm, "");
```

Comment:
🪄

Reply:
I'm not very proud of this TBH.. it's probably a temporary solution... Angular (I think) was [taking care of this](https://github.com/cernopendata/opendata.cern.ch/pull/3015/files#diff-f669327ae6143a58c0da8767f3f3b62263e36cbab79c346f8965380fc8109546L27), otherwise, HTML tags are visible, and since we trim the result is not safe to actually print it as HTML.

Reply:
It worked well, I've checked a few pages of search and I didn't find any inconsistency. In the end is just one line, well isolated and what it does is well known, so not a big deal.